assets/src/blocks/HubspotForm/HubspotFormSidebar.js

```diff
@@ -0,0 +1,112 @@
+import { CheckboxControl, PanelBody, PanelRow, Button, TextControl } from '@wordpress/components';
+import { MediaUpload, MediaUploadCheck, InspectorControls } from '@wordpress/block-editor';
+import { URLInput } from '../../components/URLInput/URLInput';
+import { useToAttribute } from './hooks/useToAttribute';
+
+const { __ } = wp.i18n;
+
+const PanelRowWraper = ({ labelText, helpText, children }) => {
+  return (
+    <div className='panel-row-wrapper'>
+      {labelText && <label className='panel-row-label-text'>{labelText}</label>}
+      {helpText && <span className='panel-row-help-text'>{helpText}</span>}
+      {children}
+    </div>
+  )
+}
+
+export const Sidebar = ({
+  ctaLink,
+  ctaNewTab,
+  blockBackgroundImageId,
+  enableCustomHubspotThankyouMessage,
+  hubspotThankyouMessage,
+  setAttributes,
+}) => {
+  const [ toAttribute ] = useToAttribute(setAttributes);
+
+  const onSelectImageHandler = ({ id, url }) => {
+    if(url && id) {
+      setAttributes({
+        blockBackgroundImageId: id,
+        blockBackgroundImageUrl: url,
+      });
+    }
+  };
+
+  const onRemoveImageHandler = () => {
+    setAttributes({
+      blockBackgroundImageId: null,
+      blockBackgroundImageUrl: null,
+    });
+  }
+
+  const getImageOrButton = (openEvent) => {
```

Comment:
Does this function name still makes sense? It doesn't seem to ever render an image 🤔